README.md

```diff
@@ -71,7 +71,27 @@ cd /workspace
 git clone https://github.com/CiscoTestAutomation/xpresso
 ```
 
-**2. Initializations**
+**2.1. Initializing using setup script**
+
+Before running the setup script
+ Make sure `xpresso_admin` password is the same across all these files
```

Comment:
Can we do this step in the script? It would make it easy as possible

Reply:
add a warning to follow 2.2 steps if they want this configured. so if they want non default passwords.. follow 2.2. It makes sense but this warning feels like it could throw someone off.